VCFv4.3.tex

```diff
@@ -219,7 +219,7 @@ \subsubsection{Assembly field format}
 \subsubsection{Contig field format}
 \label{sec-contig-field}
 It is recommended for VCF, and required for BCF, that the header includes tags describing the contigs referred to in the file.
-The structured \texttt{contig} field must include the ID attribute and typically includes also sequence length, MD5 checksum, URL tag to indicate where the sequence can be found, etc.
+The structured \texttt{contig} field must include the ID attribute and typically includes also sequence length, MD5 checksum of the sequence as defined in the \href{https://github.com/samtools/hts-specs/blob/master/refget.md}{Refget specification}, URL tag to indicate where the sequence can be found, etc.
```

Comment:
Generally the direction of these changes looks good to me! I'm especially puzzled as to why examples had all the same length and hash to start with. Thanks for noticing & correcting.

I have slight concerns about linking to an external specification (Refget); nothing that can't be addressed though. In particular, I suggest:
* Explicitly specifying the version of the specification when linking (1.0.0 in this case);
* Linking to a permalink / frozen commit, not a live branch;
* Linking to a specific section of the page (`#checksum-calculation`);
* Making the specification a footnote instead of an inline link.

And finally, I think it's a good idea to also just duplicate the actual guidance here in the specification text as well (while still keeping the footnote linking to Refget), especially seeing it's just one sentence long:

> When calculating the checksum for a sequence, all non-base symbols (\n, spaces, etc) must be removed and then uppercase the rest. The allowed alphabet for checksum calculation is uppercase ASCII (`0x41`-`0x5A` or `A-Z`).

This makes things perfectly explicit while still keeping the provenance / alignment with where this guidance originally comes from.